### DISCUSSION TITLE
Hardware recommendation with large storage 2TB or more.

### DISCUSSION THREAD

**Discussion by jlk-droid** [blakeblackshear/frigate#14050] [Ask A Question]
Hi I’m new to Frigate NVR, so I have some question.

My situation is that I got an old house without ISP, so the setup will be a stand alone.
I will need 4-6 camera with a recording rate of 15 fps. There should star recording by activities in zone.
**Important** I will prefer to have a large storage, so I can keep my recording for a while. Weeks-month.

I was looking for a Beelink EQ12 N100, with a single Coral TPU M.2 A+E in WIFI slot.
I would like to add an external USB 2TB SSD drive as storage.

Would it this setup work without problems? If not can any recommend another setup (motherboard / unit).

In a year the house will get an ISP and a HA will be added on a separate unit.

Thanks if someone can help me.

**Comment by NickM-27** [comment]
Seems like that should work fine, 2TB would be enough to keep recordings for detected objects but not 1 month worth of continuous recording
